neo4j/internal/db/connection.go

```diff
@@ -79,11 +79,12 @@ type Connection interface {
 	// streams are created and the server doesn't support multiple streams. Use Buffer to force
 	// buffering before calling Reset to get all records and the bookmark.
 	Buffer(ctx context.Context, streamHandle StreamHandle) error
-	// Bookmark returns the bookmark from last committed transaction or last finished auto-commit transaction.
+	// Bookmark returns the bookmark and optionally its database from last committed transaction or last finished auto-commit transaction.
+	// The database is only populated if queries are executed with the USE clause.
```

Comment:
The database is also populated without the usage of the `USE` cause, it is a property came from cypher.

Reply:
clarified in 112731a6f954224f93cfa92771657246abcbea04 (hopefully)